extensions/amp-ad-network-gmossp-impl/0.1/gmossp-a4a-config.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @const @private {string} */
+const GMOSSP_SRC_PREFIX_ = 'https://sp.gmossp-sp.jp';
+
+/**
+ * @param {!Window} win
+ * @param {!Element} element
+ * @returns {boolean}
+ */
+export function gmosspIsA4AEnabled(win, element) {
+  const src = element.getAttribute('src');
+  return !!element.getAttribute('data-use-a4a') && !!src &&
```

Comment:
Use src/string#startsWith to be consistent.  Note that startsWith is case sensitive.